crates/cast/tests/cli/main.rs

```diff
@@ -1054,14 +1051,16 @@ casttest!(storage_layout, |_prj, cmd| {
 casttest!(balance, |_prj, cmd| {
```

Comment:
this test is malformed, .args() doesn't execute the command so the two variables are the same